assets/license-header

```diff
@@ -0,0 +1,15 @@
+<https://github.com/radicle-dev/<project name>>
+
+<project name> is Copyright © 2019-2020 The Radicle Foundation <hello@radicle.foundation>
```

Comment:
Regarding the `2020`, is this something we will need to bump every year? as in, will we have a PR on every Jan 1 updating it to the new year?

Reply:
cc @angedupre

Reply:
If you can do that, that sounds like great plan :)

Reply:
I'll leave a note in the recommendations about this. I guess we should have a script to do this as well.

Reply:
Acc. to [Wikipedia](https://en.wikipedia.org/wiki/Copyright_notice#Form_of_notice_for_visually_perceptible_copies), US law says it should be the year of first publication. So, unless there are major copyrightable modifications, it doesn't have to be updated. And vice versa: if the file gets substantially modified, add another year, like `1996, 2032`.